macros/sql/get_query_results_as_single_value.sql

```diff
@@ -0,0 +1,29 @@
+{% macro get_query_results_as_single_value(query, row_position=0, column_position=0) %}
+    {{ return(adapter.dispatch('get_query_results_as_single_value', 'dbt_utils')(query, row_position, column_position)) }}
+{% endmacro %}
+
+{% macro default__get_query_results_as_single_value(query, row_position=0, column_position=0) %}
+
+{# This macro returns the (row_position, column_position) record in a query as a string #}
+    {%- set nth_row = row_position -%}
+    {%- set nth_column = column_position -%}
```

Comment:
These feel redundant - any specific reason you're renaming the variables two lines under where they were first created? I don't mind whether you name it nth_column or column_position, but I think it should be consistent the whole way through. 

This is of course dependent on the result of our discussion above about whether these arguments even need to exist or should always be `0`.

Reply:
Redundant, and no longer needed based on the decision above. Removing!